thrust/device_vector.h

```diff
@@ -175,7 +175,6 @@ template<typename T, typename Alloc = thrust::device_allocator<T> >
      */
     template<typename OtherT, typename OtherAlloc>
     __host__ explicit
-    __device__
```

Comment:
Do we know why this `__device__` was here?

Reply:
I have no idea. The commit that added it was vague (something like "update from github"), but since `explicit` appears in between host/device, it looks like it was unintentional.